orm/models.py

```diff
@@ -156,13 +156,22 @@ def select_related(self, related):
             select_related=related,
         )
 
+    async def exists(self) -> bool:
+        expr = self.build_select_expression()
+        expr = sqlalchemy.exists(expr).select()
```

Comment:
Should we be using `.scalar()`? https://stackoverflow.com/questions/7646173/sqlalchemy-exists-for-query

Reply:
I wasn't able to get `.scalar()` to work, because it must be used with SQLAlchemy's `session.query()` which we don't seem to have access to through orm/Databases.

I pushed a fix which hopefully clarifies how the return value from `.fetch_one()` is used. :-)

Reply:
Right yup. We probably want a `database.fetch_value()` which defaults to just returning the first value in the row.
With `database.fetch_value(column=...)` if you want to return the scalar value of some *other* column.

Should we get #17 and #18 in first, and then make the feature fix in databases (then come back here and use it) or go the other way around, and add it to databases first?

Reply:
Okay. I think we can afford coming back to this when databases adds support for such a .fetch_value() method. 👍